server/src/main/java/org/apache/seata/server/controller/ClusterController.java

```diff
@@ -117,6 +118,12 @@ public void watch(
             @RequestBody Map<String, Object> groupTerms,
             @RequestParam(defaultValue = "28000") Integer timeout) {
         context.setAsync(true);
+
+        if (Version.isAboveOrEqualVersion250(Version.getCurrent())) {
```

Comment:
I think the logic for determining whether the server is version 2.5 should not be on the server side, but should be on the client side. The client should adopt the corresponding connection method to the server based on the server's version, and the server will naturally know whether the client is using HTTP/2 requests.